xmipptomo/protocols/protocol_phantom_subtomo.py

```diff
@@ -58,64 +60,88 @@ class XmippProtPhantomSubtomo(EMProtocol, ProtTomoBase):
     def _defineParams(self, form):
         form.addSection(label='Input')
         form.addParam('option', EnumParam,
-                      choices=['Simulate tilt Series', 'Applying Missing wedge', 'creating a phantom'], default=0,
+                      choices=['Import volume', 'creating a phantom'], default=0,
```

Comment:
I'd use the same style:
Import volume, Create phantom

Or
Importing a volume, Creating a phantom

I vote for 1st.